ModuleChangelog.md

```diff
@@ -1,3 +1,36 @@
+## 2.0.0
+
+Release Date: September 16, 2021
+
+#### RELEASE NOTES
+
+```
+* Added API Key form and parameter to allow for "Auto-Binding" the user during Migration
+    * If selected, both the API Key and Connect Key values must be entered
+    * At the end of migration the ADMU will search the JumpCloud Organization for the username entered and bind the user to the system
+* GUI runs of the ADMU should prompt to view the log after migration
+* The migration user's registry hive is saved as a unique filename (ex: NTUSER_original.DAT is now NTUSER_original_yyyy-mm-dd-HHMMSS)
+* The 2.0.0 version of the tool no longer includes the Microsoft User State Migration Tool. Priror versions of the tool should be used if it's necessary to copy data from one profile to another.
+    * The default behavior of the 2.0.0 tool is to convert accounts with what was previously the `ConvertProfile` parameter.
```

Comment:
This is a breaking change for the CLI the convertProfile param does not exist for the tool and is not default behavior